spec/TotalCalculatorSpec.js

```diff
@@ -0,0 +1,27 @@
+describe('TotalCalculator', function() {
+  var totalCalculator;
+
+  describe('calculates discounts', function() {
+    it('5% off orders over £50', function() {
+      totalCalculator = new TotalCalculator();
+      totalCalculator.subTotal = 50.01;
+      totalCalculator.orderedItems = [];
+      totalCalculator.detailsList = hipsterCoffeeDetails;
+      expect(totalCalculator.calculateTotalBill()).toEqual(47.51);
+    });
+
+    it('not 5% off orders of =< £50', function() {
+      totalCalculator = new TotalCalculator();
```

Comment:
Would it make sense to pass your orderedItems as a parameter to your TotalCalculator when you create it?